devito/dse/rewriters.py

```diff
@@ -406,6 +408,38 @@ def _extract_sum_of_products(self, cluster, template, **kwargs):
 
         return cluster.rebuild(processed)
 
+    @dse_pass
+    def _skewing(self, cluster, template, **kwargs):
+        """
+        A pass to perform only the basic skewing in order to have valid data
+        dependences.
+        """
+        skew_factor = configuration['skew_factor']
+        skew_dim, mapper, int_mapper = None, {}, []
+        total_int = {}
+        intervals, sub_iterators, directions = cluster.ispace.args
+
+        for dim in cluster.ispace.dimensions:
+            if isinstance(dim, SpaceDimension):
```

Comment:
the body of this loop, conceptually, should be
```
if i.dim.is_Time:
    ...
else:
    ...
```
ie, no need to be specifically a SpaceDimension. THe only special one is Time, and to be fair, I'd love to find a way such that it's not so special... but perhaps better for another day